packages/admin/admin/src/router/PromptHandler.test.tsx

```diff
@@ -109,8 +113,10 @@ test("route outside Prompt", async () => {
                         message={() => {
                             return "sure?";
                         }}
-                    />
-                    <Link to="/bar">barLink</Link>
+                        subRoutePath="/s"
```

Comment:
Do I need to specify a dummy subRoutePath in RouterPrompt for it to work with routes on the "same" level (e.g. "/", "/bar")?

Reply:
Navigation within subRoutePath is allowed, pratical use case is the FinalForm component that allows navigation in sub routes of form (eg tabs)

Reply:
yes, here in the test it is just a dummy route

Reply:
If I had a dirty form on "/" and would try to navigate to "/bar", which is not part of the form, but a completely independent route instead, would the prompt trigger without the subRoutePath?

Reply:
> If I had a dirty form on "/" and would try to navigate to "/bar", which is not part of the form, but a completely independent route instead

exactly this (and only this) is solved by this PR

> would the prompt trigger without the subRoutePath?

without, yes. That is basically a fallback to the non-soubrute behaviour. The Prompt is at / then and /bar is a sub route.

Reply:
Thanks, now I understand it!